services/contract_call_local.proto

```diff
@@ -132,6 +132,31 @@ message ContractFunctionResult {
      * EVM address described above.)
      */
      google.protobuf.BytesValue evm_address = 9;
+
+    /**
+     * The amount of gas that was used for this call. 
```

Comment:
There's already a `gasUsed` that shows the gas used. This would be the gas limit that shows the maximum amount of gas remaining still to use, right?

Reply:
Is it supposed to represent actual gas used maybe? Related to this issue https://github.com/hashgraph/hedera-services/issues/2949.

Reply:
Steven is right this should represent the `gasLimit`.
`gasUsed` is already captured in item 5 of this proto object

Reply:
`gas` is the name of the field used in the three paired transaciton types, and I want to try and keep it the same here.  I updated the docs.

Reply:
Right, was just arguing for a doc fix here. Looks good